src/composables/useWagmiConnect.ts

```
@@ -0,0 +1,135 @@
+            } catch (error) {
+              console.error('WalletConnect connect failed:', error)
+            }
+          }
+        })
+        .catch(err => {
+          let error = t('error_connecting')
+          let _type = ToastType.Warning
+          openWalletConnectModal.value = false
+          if (
+            err.message &&
+            err.message.toLowerCase().includes('user rejected')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)